### ISSUE TITLE
More Useful Earnings Reports

### ISSUE THREAD

**Issue by SpamMusubi153** [mysteriumnetwork/node#5825]
**Is your feature request related to a problem? Please describe.**
Currently, the Mysterium Node UI "Data Report" appears to track earnings and data by session. Sessions that span multiple days have their data and earnings counted for the day the session started. Because long multi-day sessions are common, the UI-generated graphs can cause confusion or mislead node runners into thinking that their node is not actively working to transfer data.

**Describe the solution you'd like**
It would be much more helpful if the Node UI could count data and earnings towards the day the data was transferred, rather than the day the session started. It would also be helpful to have an "Average MYST Earned per Day/Week/Month" statistic.

**Comment by liamgibbins** [comment]
Basically an accounting type software, showing myst and gas paid per day, week, month, year.
Showing totals and averages.
 if I read that right?

**Comment by MarkWieczorek** [comment]
I agree that the Myst earnings report is at best very deceptive and at worst incorrect.

Consider the following: I check my node once a day and see that the amount of Myst earned in the "next auto settlement" pane is going up, but there is NO activity when looking at a 24h time span in the "earnings report". I only recently discovered, like the post above, that all earnings and data transfer are counted only for the day the session started, even though most sessions last many days.
